<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionDialog</class>
 <widget class="QDialog" name="TransactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Транзакция</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 224, 207);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Новая транзакция</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="saleBool">
      <property name="text">
       <string>Продажа</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="entranceBool">
      <property name="text">
       <string>Поступление</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="barcodeInput">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="qrButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>QR</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="itemLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Товара с таким штрих-кодом нет</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
